packages/front-end/components/Features/Rule.tsx

```diff
@@ -60,6 +62,18 @@ export const Rule = forwardRef<HTMLDivElement, RuleProps>(
       permissions.check("manageFeatures", feature.project) &&
       permissions.check("createFeatureDrafts", feature.project);
 
+    const upcomingScheduleRule = getUpcomingScheduleRule(rule);
+
+    const scheduleCompletedAndDisabled =
+      !upcomingScheduleRule &&
+      rule?.scheduleRules?.length &&
+      rule.scheduleRules.at(-1)?.timestamp !== null;
+
+    const ruleDisabled =
+      scheduleCompletedAndDisabled ||
+      upcomingScheduleRule?.enableFeature ||
```

Comment:
Question for my own knowledge. Is it possible for a rule to be currently enabled, but also have an upcoming "enableFeature" schedule?

Reply:
@bryce-fitzsimons Yep! The `enableFeature` property is a boolean, so it's possible that a rule is enabled, and the upcoming rule's `enableFeature` value is `false`. This would indicate the rule is scheduled to be disabled at a date in the future.